cross/shairport-sync/Makefile

```diff
@@ -13,7 +13,7 @@ LICENSE  = https://raw.githubusercontent.com/mikebrady/shairport-sync/$(PKG_VERS
 
 GNU_CONFIGURE = 1
 
-CONFIGURE_ARGS = --prefix=$(STAGING_INSTALL_PREFIX) --with-alsa --with-ssl=openssl --with-tinysvcmdns --without-initscript --with-piddir=$(INSTALL_DIR)/var
+CONFIGURE_ARGS = --prefix=$(STAGING_INSTALL_PREFIX) --with-alsa --with-ssl=openssl --with-tinysvcmdns --without-initscript --without-configfiles --with-piddir=$(INSTALL_DIR)/var
```

Comment:
Just a question: you're specifying `--without configfiles` here...this doesn't refer to `shairport-sync.conf`?